content/zh/docs/reference/kubernetes-api/authorization-resources/self-subject-access-review-v1.md

```diff
@@ -0,0 +1,233 @@
+---
+api_metadata:
+  apiVersion: "authorization.k8s.io/v1"
+  import: "k8s.io/api/authorization/v1"
+  kind: "SelfSubjectAccessReview"
+content_type: "api_reference"
+description: "SelfSubjectAccessReview 检查当前用户是否可以执行某操作。"
+title: "SelfSubjectAccessReview"
+weight: 2
+---
+<!--
+---
+api_metadata:
+  apiVersion: "authorization.k8s.io/v1"
+  import: "k8s.io/api/authorization/v1"
+  kind: "SelfSubjectAccessReview"
+content_type: "api_reference"
+description: "SelfSubjectAccessReview checks whether or the current user can perform an action."
+title: "SelfSubjectAccessReview"
+weight: 2
+---
+-->
+`apiVersion: authorization.k8s.io/v1`
+
+`import "k8s.io/api/authorization/v1"`
+
+## SelfSubjectAccessReview {#SelfSubjectAccessReview}
+<!--
+SelfSubjectAccessReview checks whether or the current user can perform an action.  Not filling in a spec.namespace means "in all namespaces".  Self is a special case, because users should always be able to check whether they can perform an action
+-->
+SelfSubjectAccessReview 检查当前用户是否可以执行某操作。
+不填写 spec.namespace 表示 “在所有命名空间中”。
+Self 是一个特殊情况，因为用户应始终能够检查自己是否可以执行某操作。
+
+<hr>
+
+- **apiVersion**: authorization.k8s.io/v1
+
+- **kind**: SelfSubjectAccessReview
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard list metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../authorization-resources/self-subject-access-review-v1#SelfSubjectAccessReviewSpec" >}}">SelfSubjectAccessReviewSpec</a>), required
+
+  Spec holds information about the request being evaluated.  user and groups must be empty
+-->
+  标准的列表元数据。
```

Comment:
Incorrect alignment when viewing from the preview server. This might be caused by the <!-- ... --> comment in the middle of the section. You can try to move the comment to the head of the section to correct the alignment.

Reply:
I cannot find the preview link. Could I get the link to preview? So it may be easy to correct the alignment.